api/API_VERSIONING.md

```diff
@@ -84,52 +100,43 @@ from clients (Envoy, gRPC) and a large number of control planes, ranging from si
 management servers to xDS-as-a-service offerings run by vendors. The [xDS API
 shepherds](https://github.com/orgs/envoyproxy/teams/api-shepherds) will make the decision to add a
 new major version subject to the following constraints:
-* There exists sufficient technical debt in the xDS APIs in the existing supported major version
+* There exists a major, breaking change, to the xDS APIs in the existing supported major version
```

Comment:
I don't think this needs to change in this PR. The criteria for v4 (if ever) should be discussed, but it's unrelated to adding minor versions.

Reply:
Reverted.